python/cudf/setup.py

```diff
@@ -16,6 +16,24 @@
 
 cython_files = ["cudf/**/*.pyx"]
 
+libraries = [
+    "cudf",
+    "cudf_base",
+    "cudf_ast",
+    "cudf_comms",
+    "cudf_hash",
+    "cudf_interop",
+    "cudf_io",
+    "cudf_join",
+    "cudf_merge",
+    "cudf_partitioning",
+    "cudf_reductions",
+    "cudf_replace",
+    "cudf_reshape",
+    "cudf_rolling",
+    "cudf_transpose"
+]
```

Comment:
Does the transitive linking not solve this problem? I thought the whole point was that there's still a single `libcudf` target someone can use?

Reply:
The transitive links address the problem in the C++ library and the associated tests, but Python does not detect/address this.